charts/barman/\

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```

Comment:
Why is this file called `\`? It looks like it's a dupe of `charts/barman/templates/configmap-barman-cron.yaml` or `charts/barman/templates/configmap-barman-backups.yaml`.

Reply:
whoops